manifests/globals.pp

```diff
@@ -76,14 +76,14 @@
         }
       } else {
         case $globals_php_version {
-          /^7/: {
+          /^7\.[0-9]/: {
```

Comment:
Being able to use:
`$package_prefix       = "php${globals_php_version}-"`

Because the package prefix is `php7.0-` and not `php7-`
Before that change it was hardcoded.